src/blacklist.js

```diff
@@ -0,0 +1,12 @@
+const Url = require("url");
+
+const blacklist = {
+  'www.google-analytics.com': true,
```

Comment:
When is the value ever `false`?
Why not make this an array or a set? Sets are as fast to look up as dictionaries.

Reply:
Ah yes, I forgot about new ES features. Set is what I was looking for